pkg/login/socialtest/social_connector_mock.go

```diff
@@ -22,11 +22,11 @@ type MockSocialConnector struct {
 
 // AuthCodeURL provides a mock function with given fields: state, opts
 func (_m *MockSocialConnector) AuthCodeURL(state string, opts ...oauth2.AuthCodeOption) string {
-	_va := make([]any, len(opts))
+	_va := make([]interface{}, len(opts))
```

Comment:
I think we should prefer any over `interface{}`

Reply:
I replaced `interface{}` with `any` in every file (hopefully :) )